core/src/main/res/values-de/strings.xml

```diff
@@ -819,4 +819,5 @@
   <string name="on_demand_config_setting_changed">Einstellung erfolgreich aktualisiert.</string>
   <string name="on_demand_config_stream_text">Es sieht so aus als würdest du viel streamen. Möchtest du in Listen den Streamen-Button anzeigen?</string>
   <string name="on_demand_config_download_text">Es sieht so aus als würdest du viel herunterladen. Möchtest du in Listen den Herunterladen-Button anzeigen?</string>
+    <string name="gpodnetauth_encryption_warning">Passwort und Daten sind nicht verschlüsselt!</string>
```

Comment:
Please only change the English strings. The other ones get overwritten when pulling the translations from Transifex.

Reply:
Done... waiting for it to pop up in Transifex